runtime/oti/j9port_generated.h

```diff
@@ -680,7 +680,7 @@ extern J9_CFUNC int32_t j9port_isCompatible(struct J9PortLibraryVersion *expecte
 #define j9sock_getaddrinfo_length(param1,param2) privatePortLibrary->sock_getaddrinfo_length(privatePortLibrary,param1,param2)
 #define j9sock_getaddrinfo_name(param1,param2,param3) privatePortLibrary->sock_getaddrinfo_name(privatePortLibrary,param1,param2,param3)
 #define j9sock_error_message() privatePortLibrary->sock_error_message(privatePortLibrary)
-#define j9str_ftime(param1,param2,param3,param4) OMRPORT_FROM_J9PORT(privatePortLibrary)->str_ftime(OMRPORT_FROM_J9PORT(privatePortLibrary),param1,param2,param3,param4)
+#define j9str_ftime(param1,param2,param3,param4,param5) OMRPORT_FROM_J9PORT(privatePortLibrary)->str_ftime_ex(OMRPORT_FROM_J9PORT(privatePortLibrary),param1,param2,param3,param4,param5)
```

Comment:
You are changing all the callers anyway, it's preferred to remove this and call omrstr_ftime_ex() directly.

Reply:
Will do.